### DISCUSSION TITLE
Use Polyglot in non5e systems?

### DISCUSSION THREAD

**Discussion by rainier-m** [mclemente/fvtt-module-polyglot#225] [General]
I was reading through the documentation and I wanted to use this Module for a "real world" spy themed game. How would I go about adding languages to the character sheet, system and using this module to facilitate some of the usages? I've added in languages using the module settings, but want to get them listed in skills and on the character sheet.

Example: Player A speaks English and Spanish and is eavesdropping on an NPC who speaks Russian.

**Comment by mclemente** [comment]
Create an item with a name like this: `Language (English)` and add it to your character's sheet and it'll show up on the language selector on the chat window.

You haven't informed which system you are using, so I can't tell how to add languages to the skills (or if that's even possible).

**Reply by rainier-m** [reply]
Sorry about that, Matheus! I'm using the Twilight:2000 4th Edition rules, and was tinkering with adding the languages as "Custom Languages" in the Module Settings. 

I added these in the Custom Settings:
* English, Russian, Polish, Ukranian, French, German, Italian, Swedish

Then I added an Item of type "Specialty" with the description of Language (Turkish). I don't see this in the Polyglot dropdown, but I see the custom languages.

**Reply by mclemente** [reply]
Hey, sorry for the late reply. Turkish won't show on the dropdown unless it's added as a Custom Language. It will show if you're selecting a character that has that specialty, but it'll show on lowercase. So it's better to add all the languages to the setting instead.
